frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -396,8 +393,16 @@ export default function InteractiveWaveformModal({
     }
   };
 
+  const resetStartEndTimes = () => {
+    const { start_time, end_time } = selectedJob.analysisParams;
+    setEditableStartEndTimes({
+      startTime: start_time,
+      endTime: end_time,
+    });
+  };
+
   const loadExistingData = () => {
-    // this happens very fast so not storing to react state the first call, see line 162
+    // this happens very fast so not storing to react state the first call, see line 162 (? different line now)
```

Comment:
did I write that comment? I don't really know what I meant by that

Reply:
I'm not sure who wrote it